### DISCUSSION TITLE
Add multiple windows to WindowManager

### DISCUSSION THREAD

**Discussion by jaakkokoo** [bczsalba/pytermgui#44] [Ideas]
Hello,

New to pytermgui and building a dashboard to manage multiple server information. Is it possible to add multiple windows to one window manager so that the view would look like this: 
![image](https://user-images.githubusercontent.com/40392086/161123632-796a330f-137f-4f96-933a-cbfe19778968.png)

**Comment by bczsalba** [comment]
Hey!

At the moment layouts like this are not very easy to do out of the box, but I'm pretty sure you can come up with a small algorithm to space windows out this way, but there isn't a built in way to do it ATM.

Layouts are a thing that _should_ arrive in the near-ish future though.

Thank you for your interest!

**Reply by jaakkokoo** [reply]
Hi, 

Thanks for quick reply. I will look into if I can come up with a algorithm to solve this problem

**Reply by bczsalba** [reply]
Feel free to share it if you do, might end up borrowing it in the layout implementation! :)

**Reply by bczsalba** [reply]
@jaakkokoo Not sure if this is still applicable, but the layout system was implemented a while back and might be useful to you :)
